### PR TITLE
Center Widget API and Transparency support in foreground

### PR DESCRIPTION
**Pull request by vidager**
Sample updates for the prerelease version of the 2104 SDK (to be released)

1. New manifest entry to op-in to transparency support while Game Bar is in the Foreground.
2. New API to center widget on the screen.

### CODE REVIEW

File: Samples/WidgetAdvSample/Widget1.h [hunk 1]
```diff
@@ -51,7 +53,8 @@ namespace winrt::WidgetAdvSample::implementation
         // Methods to handle updating of Text and UI
         void SetBackgroundColor();
         void SetBackgroundOpacity();
-        hstring RequestedOpacityToString();
+        void SetRequestedOpacityState();
+        hstring OpacityToString();
```
Comment (bertburtbert) [inline Samples/WidgetAdvSample/Widget1.h:57]: Do we still need this?

**Review by bertburtbert — APPROVED** [review]

File: Samples/WidgetAdvSample/Widget1.h (hunk 1 above)
Comment (vidager) [reply Samples/WidgetAdvSample/Widget1.h:57]: Nope. Good catch

**Review by totallymonkey — COMMENTED** [review]
Are you going to update (before checking in) to use a non-local nuget once we have one published with the right changes?

**Comment by vidager** [comment]
> Are you going to update (before checking in) to use a non-local nuget once we have one published with the right changes?

Before it moves to main for sure.  For this PR to 'prerelease', I will likely just update to the internal feed so we can get the CI/PR build working.

**Review by totallymonkey — APPROVED** [review]

**Pull request merged**